app/driver.go

```diff
@@ -245,6 +245,21 @@ func deployCSIDriver(kubeClient *clientset.Clientset, lhClient *lhclientset.Clie
 		return err
 	}
 
+	// Longhorn #1365, remove compatible CSI deployment
+	compatCSIAttacherName := "compatible-csi-attacher"
+	compatCSIAttacherDeployment, err :=
+		kubeClient.AppsV1().Deployments(namespace).Get(
+			compatCSIAttacherName, metav1.GetOptions{})
+
+	// Delete compatible-csi-attacher deployment if it exists
+	if err == nil {
+		logrus.Debugf("Deleting %v deployment",
+			compatCSIAttacherDeployment.Name)
+
+		err = kubeClient.AppsV1().Deployments(namespace).Delete(
```

Comment:
err is unused later.

We can print out a warning here is it returns an error.

Reply:
@yasker , the err is about "not found", do you think we should print it every time the new csi driver re-deployed?
Thanks.

Reply:
It's already found right? on line 250. Otherwise we won't hit here.

Reply:
@yasker , yes. I was thinking the err for "Get". I will update it now. Thanks.